localstack-core/localstack/plugins.py

```diff
@@ -19,6 +19,7 @@ def deprecation_warnings() -> None:
     LOG.debug("Checking for the usage of deprecated community features and configs...")
     from localstack.deprecations import log_deprecation_warnings
 
+    LOG.warning("Community change")
```

Comment:
q: what is this line doing?

Reply:
Testing the code mounting, that should not have made it into the PR, removing it :)